src/static/app/index.js

```diff
@@ -46,6 +48,7 @@ import './error/404.tag'
 global.route = route;
 global.observable = riot.observable();
 riot.mount('*')
+riot.mixin(ToastMixin);
```

Comment:
[Q]
動くなら多分どちらでもいいん気がするんだけど mount('*') したあとに
mixin の読み込みで大丈夫なのかが気になる 🤔

mount した tag の中から mixin を使っているので、一瞬読み込み先がないとかにならないのだろうか、、、

```suggestion
riot.mixin(ToastMixin);
riot.mount('*')
```

Reply:
ちょっと実験してみた結果、 `今の利用箇所的には問題なかったけど、globalなmixinは先にしておくべき` って感じだった！
ナイス指摘です 👍 

**実験したこと**
Q. そもそもここの `riot.mount(*)` ではどのtagがmountされるのか？
A. ['loading', 'result-view', 'su-toast', 'su-confirm']の4つ
※ `let tags = riot.mount('*');console.log(tags.map(t => t.root.localName))` で出力できる

これは `dbflute-intro/src/static/index.htmlで記述されているカスタムタグ` かつ `import文で宣言されているカスタムタグ`
したがって、以下はindex.htmlに記述されているがmountされない
- network-error-view -> 同名のカスタムタグが存在しない
- side-menu, content -> 同名のカスタムタグが存在しない（が、そのあとセレクターを指定する形でmountされる）

Q. すでにmountされたカスタムタグにmixinを後付けできるのか？
A. できない

```javascript
// mixinを後に宣言
const tags = riot.mount('*');
riot.mixin(ToastMixin);
console.log(tags.map(t => `${t.root.localName} tag has successToast is ${!!t.successToast}`))
// → (4) ['loading tag has successToast is false', 'result-view tag has successToast is false', 'su-toast tag has successToast is false', 'su-confirm tag has successToast is false'] 

// mixinを先に宣言
riot.mixin(ToastMixin);
const tags = riot.mount('*');
console.log(tags.map(t => `${t.root.localName} tag has successToast is ${!!t.successToast}`))
// → (4) ['loading tag has successToast is true', 'result-view tag has successToast is true', 'su-toast tag has successToast is true', 'su-confirm tag has successToast is true']
```

Reply:
> Q. そもそもここの riot.mount(*) ではどのtagがmountされるのか？
> A. ['loading', 'result-view', 'su-toast', 'su-confirm']の4つ

え、全タグじゃなかったのか、勘違いしてた、なるほどなるほど 🤔

Reply:
> Q. すでにmountされたカスタムタグにmixinを後付けできるのか？
> A. できない。mixinを実行した後にmountされたタグのみ追加される模様

おおーやっぱりそうか、実験ありがとー ✨